src/main/java/commands/CreateFile.java

```diff
@@ -0,0 +1,16 @@
+package commands;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CreateFile {
+    public static void createFile(File f) {
+        try {
+            if(f.createNewFile()) {
+                System.out.println("File Created");
+            }
+        } catch (IOException e) {
+            System.out.println("Error occurs");
```

Comment:
Perhaps you can print out what kind of error it is instead of using a generic "error occurs"